kingpin/actors/rightscale/server_array.py

```diff
@@ -978,24 +985,27 @@ def _wait_until_healthy(self, array, sleep=60):
         # Get the current min_count setting from the ServerArray object, or get
         # the min_count from the count number supplied to the actor (if it
         # was).
-        min_count = int(self._options.get('count', False))
+        min_count = float(self.option('count'))
+        wait_pct = float(self.option('wait_pct'))
         if not min_count:
             min_count = int(array.soul['elasticity_params']
                             ['bounds']['min_count'])
 
+        min_wait = int(math.ceil(min_count * (wait_pct / 100.0)))
```

Comment:
Thanks! I was struggling to find it. Hate the wait term. Maybe rename the option to target_pct also?